deepchem/utils/molecule_graph.py

```diff
@@ -0,0 +1,140 @@
+from typing import Optional, Iterable
+import numpy as np
+
+
+class MoleculeGraphData(object):
+  """MoleculeGraphData class
+
+  This data class is almost same as `torch_geometric.data.Data
```

Comment:
Could we add a function to convert this graph to `torg_geometric.data.Data`?

Reply:
I added, but I don't check working well